### PR TITLE
Feature/spread eth frontend

### LINKED ISSUE #74: ETH diversification strategy
# Problem

Whip can currently be used to backtest USDC, however [Radicle are considering diversification into ETH](https://radicle.community/t/make-eth-investment-with-treasury-usdc-funds/2882/11) using their existing USDC holdings. Whip cannot currently support ETH backtests.

Image

# Solution

Implement the ETH strategy

### LINKED ISSUE #75: Remove placeholder strategies
# Problem

Whip currently shows four strategies, diversify into USDC, diversify into ETH, debt funding with UMA's range token, and Opyn's Squeeth crab strategy.

Image

The USDC strategy is the only functional strategy, with the other three representing placeholders. These placeholders are useful for the development team but may confuse users when they click them. 

# Solution

Remove/hide inactive strategies

### PR DESCRIPTION
**Pull request by lajarre**
Fixes #74
Fixes #75

Did some slight changes to the frontend @vaughnmck , testable with the link below (as soon as it appears).

### CODE REVIEW

File: backend/app/spread/actions.py [hunk 1]
```diff
@@ -30,25 +31,36 @@ def update_balances_with_spread(
     if token_to_divest_from == spread_token_symbol:
         return resized_balances
 
-    zeroes_series = make_zeroes(start, end)
+    def make_spread_token_unit_series():
```
Comment (acemasterjb) [inline backend/app/spread/actions.py:34]: Can you explain this function? I can tell that it returns a `Series`, and that it divides the given price TS of the spread token by the initial price in the token's TS but I can't really tell why you're doing this. What is a unit series, and what does it achieve?

File: backend/app/spread/actions.py [hunk 2]
```diff
@@ -30,25 +31,36 @@ def update_balances_with_spread(
...
         ) / 100.0
 
-    if spread_token_symbol in resized_balances.balances:
-        spread_additional_token_balance = spread_additional_token_balance.add(
-            resized_balances.balances[spread_token_symbol], fill_value=0
+    spread_token_additional_usd_balance = (
+        divest_token_start_usd_balance
+        * spread_percentage
+        / 100
+        * make_spread_token_unit_series()
```
Comment (acemasterjb) [inline backend/app/spread/actions.py:54]: Similar to my comment above. I get the sense that this unit series is used to elegantly get the USD value of the treasury's "spread token" balance post-spread action, but it doesn't immediately become clear to me.

Honestly, this implementation is very elegant, but maybe this function overall could be broken down to multiple functions at a later refractor and maybe the variable themselves could be named better for more clarity. Some type hints for these variables would also be helpful.

**Comment by vaughnmck** [comment]
@lajarre link is inactive

File: backend/app/spread/actions.py (hunk 1 above)
Comment (lajarre) [reply backend/app/spread/actions.py:34]: It's the series of usd balances if you hold a constant balance of 1 token (like, 1 ETH or 1 USDC).

I'll try changing the name and I'll definitely add a comment.

File: backend/app/spread/actions.py (hunk 2 above)
Comment (lajarre) [reply backend/app/spread/actions.py:54]: I'll try refactoring a bit.

Comment (lajarre) [reply backend/app/spread/actions.py]: I hope these explanations help.

Comment (acemasterjb) [reply backend/app/spread/actions.py]: Yup, that does it for me! With that I think this PR is ready to merge.

**Comment by acemasterjb** [comment]
> @lajarre link is inactive

@vaughnmck Link should be up now.

**Comment by vaughnmck** [comment]
Looks like allocation isn't being updated:
```
"ETH":{"allocation":0.0,"volatility":0.03307142314055721,"metrics":{...
```

**Comment by lajarre** [comment]
> Looks like allocation isn't being updated:

It should be fixed, can you please double-check @vaughnmck ? Thanks

**Comment by vaughnmck** [comment]
Yes. Seems to be working now:

Screenshot 2022-08-03 at 11 00 35

No further comments from me on this issue.

**Pull request merged**